app/views/events/show.html.erb

```diff
@@ -68,10 +68,16 @@
   <% if event_status_open?(@event) || event_status_pending?(@event) %>
     <% if can_sign_up_online?(@event) %>
       <h2>How to attend</h2>
-      <p>
-        To attend this event, you must register for a place. Once registered, you will receive log-in information and joining instructions via email.
-        Please note: to access this event you will require a laptop/desktop pc and be using chrome as your browser.
-      </p>
+      <% if @event.is_online %>
+        <p>
+          To attend this event, you must register for a place. Once registered, you will receive log-in information and joining instructions via email.
+          Please note: to access this event you will require a laptop/desktop pc and be using chrome as your browser.
```

Comment:
Should PC and Chrome be capitalised?

Reply:
Actually not sure the sentence makes sense. Maybe:

Please note: in order to access this event, you will need to be using a laptop/desktop PC with Google Chrome as your browser.

Reply:
@MylesJarvis any preference? I haven't changed this text but we can update it while we're adding the new text anyway

Reply:
Oh yeah, sorry!

Reply:
@ethax-ross happy for that to be amended!